src/commonStyles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$background-footer-main-color: #59ABB0;
```

Comment:
It's going to be used not only in footer, it's just main theme color. Can be named $main-theme-color